docs/docs/guides/core-types/datasets.md

```diff
@@ -175,17 +202,39 @@ You can edit and delete existing `Dataset`s from the **Datasets** tab in the UI.
 ### Delete a `Dataset`
 
 1. Navigate to the Weave project containing the `Dataset` you want to edit.
+
 2. From the sidebar, select **Datasets**. Your available `Dataset`s display.
-3. In the **Object** column, click the name and version of the `Dataset` you want to delete. A pop-out modal showing `Dataset` information like name, version, author, and `Dataset` rows displays.
 
-4. In the upper right-hand corner of the modal, click the trash can icon. 
+3. In the **Object** column, click the name and version of the `Dataset` you want to delete. A pop-out modal showing `Dataset` information like name, version, author, and `Dataset` rows displays.
 
-   ![`Dataset` UI - Delete a `Dataset` icon.](./imgs/dataset-trashcan.png)
+4. In the upper right-hand corner of the modal, click the trashcan icon. 
 
    A pop-up modal prompting you to confirm `Dataset` deletion displays. 
 
    ![`Dataset` UI - Confirm deletion modal.](./imgs/datasetui-delete-modal.png)
 
 5. In the pop-up modal, click the red **Delete** button to delete the `Dataset`. Alternatively, click **Cancel** if you don't want to delete the `Dataset`. 
 
-   Now, the `Dataset` is deleted, and no longer visible in the **Datasets** tab in your Weave dashboard.
\ No newline at end of file
+   Now, the `Dataset` is deleted, and no longer visible in the **Datasets** tab in your Weave dashboard.
+
+### Add a new example to a `Dataset`
+
+1. Navigate to the Weave project you want to edit.
+
+2. In the sidebar, select **Traces**.
+
+3. Select one or more calls with `Datasets` for which you want to create new examples.
+
+4. In the upper right-hand menu, click the **Add selected rows to a dataset** icon (located next to the trashcan icon). Optionally, toggle **Show latest versions** to off to display all versions of all available datasets.
+
+5. From the **Choose a dataset** dropdown, select the `Dataset` you want to add examples to. Options to **Configure field mapping** will display.
+
+6. (Optional) In **Configure field mapping**, adjust the mapping of fields from your calls to the corresponding dataset columns.
+
+7. Once you've configured field mappings, click **Next**. The **Add examples to a dataset** screen appears.
```

Comment:
I don't think we want to refer to the final page of the drawer as the `Add examples to a dataset screen`, it's just another step in the drawer in which you can make final adjustments to the dataset and then confirm.

Reply:
I think it will help if I add some text that just says "edit and confim" above the editor, beneath the drawer title